docs/hyperparameter_search.rst

```diff
@@ -0,0 +1,136 @@
+.. inclusion-marker-start-do-not-remove
+Distributed Hyperparameter Search
+=================================
+
+Horovod's data parallelism training capabilities allows you to scale out and speed up the workload of training a deep learning model. However, distributed training often does not exhibit linear scaling in accuracy improvement - i.e., simply using 2x more workers does not necessarily mean the model will obtain the same accuracy in 2x less time.
+
+To address this, you will often need to re-tune hyperparameters when training at scale, as many hyperparameters exhibit different behaviors at larger scales.
+
+To address this issue, Horovod offers a Ray Tune integration to enable parallel hyperparameter tuning with distributed training.
+
+Ray Tune is an industry standard tool for distributed hyperparameter tuning. Ray Tune includes the latest hyperparameter search algorithms, integrates with TensorBoard and other analysis libraries, and natively supports distributed training. The Ray Tune + Horovod integration leverages the underlying Ray framework to provide a scalable and comprehensive hyperparameter tuning setup.
+
+**By the end of this guide, you will learn:**
+* How you can set up Ray Tune and Horovod to tune your hyperparameters
```

Comment:
I think you need a blank line before the list begins

Reply:
done!